scripts.js

```diff
@@ -0,0 +1,172 @@
+    let charts = document.getElementsByClassName("chart-edu")[0];
+    charts.innerHTML += '<div class="event-chart"> \
+							<div class="arrow-right"></div> \
+							<div class="line-dot-horizon"> \
+							</div> \
+							<div class="chart-content"> \
+								<h4 class="event-name">(<span>2010 - 2015</span>) DEF UNIVERSITY</h4> \
+								<p>Student</p> \
+								<p>Lorem ipsum dolor sit amet, consectetuer adipiscing elit,\
+								 sed diam nonummy nibh euismod tincclassunt</p> \
+							</div> \
+						</div>';
+}
+
+function addProjectChart() {
+
+    let charts = document.getElementsByClassName('sec-graphic')[0];
+    let numTypeChart = 9;
+    let horOrVerClass, image;
+    let numExistedChart = charts.childElementCount;
+    let typeNewChart = numExistedChart % numTypeChart;
+
+    //set rec-vertical or rec-horizon
+    if (typeNewChart == 2 || typeNewChart == 6) {
+        horOrVerClass = 'rec-vertical';
+    } else {
+        horOrVerClass = 'rec-horizon';
+    }
+
+    //sec image
+    if (typeNewChart == 2 || typeNewChart == 7) {
+        image = '2222.png';
+    } else {
+        image = '1111.png'
+    }
+
+    // xóa rec-btn
+    document.getElementsByClassName('rec-btn')[0].remove();
+
+    charts.innerHTML += '<div class="row justify-content-center' + ' rec' + typeNewChart + ' ' + horOrVerClass + '">\
+							<div><img src="imgs/' + image + '"></div> \
+						</div> \
+						<div class="rec-horizon rec-btn"> \
+							<div class="row justify-content-center wrap-btn"> \
+								<div class="btn btn-project" onclick="addProjectChart()"> \
+									<span>Add project</span> \
+									<i class="fa fa-plus"></i> \
+								</div> \
+							</div> \
+						</div>';
+}
+
+$(document).on('ready', function () {
+
+    //slick references
+    $(".regular").slick({
+        dots: true,
+        infinite: true,
+        slidesToShow: 1,
+        slidesToScroll: 1,
+        autoplay: false,
+        autoplaySpeed: 7000
+    });
+
+    //slick add slide
+    $('.btn-add-ref').on('click', function () {
+        let newRef = '<div>\
+						<div class="row justify-content-center chart-content-ref">\
+							<div class=" avatar-ref avatar"></div>\
+							<div class="col-md sec-quotes">\
+								<span class="quotes quote-open"> “</span>\
+								<span>Lorem ipsum dolor sit amet, consectetuer adipiscing elit, \
+								sed diam nonummy nibh euismod tincclassunt ut laoreet dolore magna aliquam erat volutpat. \
+								Ut wisi enim ad minim veniam, quis nostrud exerci tation ullamcorper suscipit lobortis nisl \
+								ut aliquip ex ea commodo consequat. Duis autem vel eum iriure dolor in hendrerit in</span>\
+								<span class="quotes quote-end">”</span>\
+							</div>\
+						</div>\
+					</div>';
+        $('.regular').slick('slickAdd', newRef);
+    });
+
+    //Set clicked event for language choosen.
+    $('.language span').click(function () {
+        $('.language span').removeClass('lan-choosed');
```

Comment:
$(this)

Reply:
cái này là em xóa hết class lan-choosed ở thẻ .language span, nếu dùng this nó chỉ xóa ở một thẻ thôi